src/test/java/uk/gov/justice/digital/delius/transformers/OffenderTransformerTest.java

```diff
@@ -1,8 +1,20 @@
 package uk.gov.justice.digital.delius.transformers;
```

Comment:
IntelliJ suggested a ton of changes in this file that  I accepted, sorry, totally unrelated to my change which is at the end